apps/backend/src/libs/modules/controller/libs/types/controller-route-parameters.type.ts

```diff
@@ -7,6 +9,7 @@ type ControllerRouteParameters = {
 	handler: APIHandler;
 	method: HTTPMethod;
 	path: string;
+	preHandler?: preHandlerHookHandler | preHandlerHookHandler[] | undefined;
```

Comment:
```suggestion
	preHandlers?: preHandlerHookHandler | preHandlerHookHandler[] | undefined;
```
We can have multiple hooks